__tests__/app.test.js

```diff
@@ -0,0 +1,41 @@
+const app = require("../app.js");
+const request = require("supertest");
+const db = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+const data = require("../db/data/test-data");
+
+beforeEach(() => {
+  return seed(data);
+});
+afterAll(() => {
+  return db.end();
+});
+
+
+  describe("/api/anything", () => {
+    test("GET: 404 - responds with msg: End point not found", () => {
+      return request(app)
+        .get("/anything")
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe("End point not found");
+        });
+    });
+ 
+  
+  describe("/api/topics", () => {
+    test("GET:200 - responds with all topics", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+          body.topics.forEach((topic) => {
+            expect(Object.keys(topic).length).toBe(2);
```

Comment:
expecting there to be 2 keys makes this test rigid, if you decided to alter the topics table and add more properties you would have to come back and change this test